furniture_store/demo/website_view.xml

```diff
@@ -0,0 +1,538 @@
+                                        <div class="s_media_list_body col-lg-6">
+                                            <h3>
+                                                Summer Essentials
+                                            </h3>
+                                            <p>
+                                                The right summer essentials can make the difference between a good time and a stand-out great time. Discover our selection for everyday wear during the warm summer months.
+                                            </p>
+                                            <a href="#" class="btn btn-primary mb-2">Read more</a>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </section>
+                    <section
+                        class="s_call_to_action o_cc pt120 pb104 o_cc4 o_colored_level"
+                        data-oe-shape-data='{"shape":"web_editor/Floats/02","flip":[]}'
+                        data-snippet="s_call_to_action"
+                        data-name="Call to Action"
+                        style="background-image: none;"
+                    >
+                        <div class="o_we_shape o_web_editor_Floats_02 o_footer_extra_shape_mapping" />
+                        <div class="container">
+                            <div class="row">
+                                <div class="pb16 col-lg-12 o_colored_level">
+                                    <h3 style="text-align: center;">
+                                        Be yourself, create your own design
+                                    </h3>
+                                    <p style="text-align: center;">
+                                        Discover the new designs.
+                                    </p>
+                                </div>
+                                <div class="pt8 col-lg-12 o_colored_level">
+                                    <p style="text-align: center;">
+                                        <a class="btn btn-primary btn-lg mb-2 btn-lg" href="/contactus">
+                                            Discover it
+                                        </a>
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                    </section>
+                </div>
+            </t>
+        </t>
+    </template>
+    <record id="website_page_aboutus" model="ir.ui.view">
+        <field name="name">About Us</field>
+        <field name="website_id" ref="website.default_website" />
+        <field name="key">website.about-us</field>
+        <field name="type">qweb</field>
+        <field name="arch" type="xml">
+            <t name="About Us" t-name="website.about-us">
```

Comment:
I wonder if should keep this version of "about us". There is no text, only pictures... I really don't see the value.